app/components/search/SearchResultsContainer.jsx

```diff
@@ -49,13 +70,27 @@ No results have been found for
             {searchResults.query.length > 0
                 && <h1>{searchResults.query}</h1>
             }
-            <Filtering />
+            <Filtering
+              setMapButton={setMapButton}
+              getMapButton={getMapButton}
+              isMobile={isMobile}
+            />
           </div>
-          <SearchTable
-            hits={hits}
-            page={searchResults.page}
-            hitsPerPage={searchResults.hitsPerPage}
-          />
+          { (getMapButton && isMobile) ? (
+            <SearchMap
+              hits={hits}
+              page={searchResults.page}
+              hitsPerPage={searchResults.hitsPerPage}
+              isMobile
+            />
+          ) : (
+            <SearchTable
+              hits={hits}
+              page={searchResults.page}
+              hitsPerPage={searchResults.hitsPerPage}
+            />
+          )
+          }
```

Comment:
I think this conditional will completely go away if you make more of the media queries in CSS, like I recommended, but I wanted to point out a general tip about stuff like this. Since the result of the conditional is to just set or unset a single prop on the component, you don't actually need the ternary expression around it and you can instead just pass a boolean value to `isMobile`. For example, this could just be written as:

```jsx
<SearchMap
  hits={hits}
  page={searchResults.page}
  hitsPerPage={searchResults.hitsPerPage}
  isMobile={getMapButton && isMobile}
/>
```

Reply:
I see, thank you for the detailed input! I'll take a stab at implementing this using media queries if I get the time. Until then I'll leave this ticket open again on Clubhouse in case anyone else is able to get the job done before me. I'll be sure to contact you regarding any questions I have on implementing this. Cheers!

Reply:
Cool, thanks! Let me know if you ever wanted to spend a bit of time chatting about this and I'd be happy to help out.